src/stylesheets/Responsive.css

```diff
@@ -6,4 +6,63 @@
   .sub-header {
     font-size: 2rem;
   }
+
+  .portal-wrapper {
+    margin-left: auto;
+    margin-right: auto;
+    width: 50%;
+  }
+
+  .small-grid-outer {
+    bottom: 15rem;
+    right: 5rem;
+  }
+}
+
+@media only screen and (min-width: 1024px) {
+  .small-grid-outer {
+    display: none;
+  }
+
+  .large-grid-outer {
+    display: block;
+    bottom: 5rem;
+    right: 5rem;
+  }
+
+  .box-front {
+    padding: 5rem;
+  }
+}
```

Comment:
I can tell you really thought about your media queries and keeping them as simple as possible without having to make too many changes.